buildSrc/src/main/kotlin/Ci.kt

```diff
@@ -1,8 +1,7 @@
 object Ci {
-
    // this is the version used for building snapshots
    // .GITHUB_RUN_NUMBER-snapshot will be appended
-   private const val snapshotBase = "1.0.0"
+   private const val snapshotBase = "2.0.0"
```

Comment:
Just wanted to make a call out that I changed this to version 2.0.0 here. Please let me know if this should be reverted or changed to something else. I'd be happy to make the change.

Reply:
I think it's a good idea to go for 2.0 👍🏼

Reply:
Excellent! I'll leave that as it is. I updated the sonatype url. Do you need me to update the README as well to mention something about v2? Is there anything else you need before we could release v2 here?

Reply:
Updating readme would be appreciated. I think I can take care of the rest. 😃

Reply:
Excellent! I have updated the README to include 2.0.0 release notes 🙂